visual_behavior/visualization/ophys/population_summary_figures.py

```diff
@@ -472,7 +472,7 @@ def plot_tuning_curve_heatmap(df, vmax=0.3, sup_title=None, title=None, ax=None,
 
 
 def plot_pref_stim_responses(df, vmax=0.3, colorbar=False, ax=None, save_dir=None, folder=None, interval_sec=2,
-                             use_events=False, window=[-4, 4]):
+                             use_events=False, window=[-4, 4], frame_rate=31.):
```

Comment:
should the frame rate be hardcoded?

Reply:
good catch. i think this is a holdover from an old way of doing things. ill update it.

Reply:
im going to leave this for now because its an old function that needs replaced anyway. not worth the effort to reconfigure it.